packages/mask/src/components/shared/PersonaListDialog.tsx

```diff
@@ -0,0 +1,324 @@
+import { WalletMessages } from '../../plugins/Wallet/messages'
+import { useRemoteControlledDialog } from '@masknet/shared-base-ui'
+import { makeStyles, useCustomSnackbar } from '@masknet/theme'
+import { activatedSocialNetworkUI } from '../../social-network'
+import { PluginNextIDMessages } from '../../plugins/NextID/messages'
+
+const useStyles = makeStyles()((theme) => {
+    return {
+        root: {
+            width: 384,
+            height: 386,
+            padding: theme.spacing(1),
+            background: theme.palette.maskColor.bottom,
+        },
+        content: {
+            padding: theme.spacing(0, 2, 2, 2),
+            scrollbarWidth: 'none',
+            '&::-webkit-scrollbar': {
+                display: 'none',
+            },
+        },
+        header: {
+            background: theme.palette.maskColor.bottom,
+        },
+        nickname: {
+            fontSize: 16,
+            lineHeight: '20px',
+            color: theme.palette.maskColor.main,
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+            whiteSpace: 'nowrap',
+        },
+        fingerprint: {
+            fontSize: 12,
+            lineHeight: '16px',
+            color: theme.palette.maskColor.second,
+        },
+        copyIcon: {
+            color: 'red',
+        },
+        dot: {
+            display: 'inline-block',
+            background: '#2DDF00',
+            borderRadius: '50%',
+            width: 7,
+            height: 7,
+            position: 'absolute',
+            left: '77.62%',
+            right: '5.84%',
+            top: '5.84%',
+            bottom: '77.62',
+            border: `1px solid ${theme.palette.maskColor.bottom}`,
+        },
+    }
+})
+
+interface PersonaListProps {
+    open: boolean
+    onClose(): void
+}
+
+interface PersonaNextIDMixture {
+    persona: PersonaInformation
+    proof: BindingProof[]
+    avatar?: string
+}
+
+export const PersonaListDialog = ({ open, onClose }: PersonaListProps) => {
+    const { t } = useI18N()
+    const { classes } = useStyles()
+    const [, copyToClipboard] = useCopyToClipboard()
+    const { showSnackbar } = useCustomSnackbar()
+    const [selectedPersona, setSelectedPersona] = useState<PersonaNextIDMixture>()
+    const currentPlatform = activatedSocialNetworkUI.configuration.nextIDConfig?.platform as NextIDPlatform | undefined
+
+    const [, handleVerifyNextID] = useNextIDVerify()
+    const currentProfileIdentify = useLastRecognizedIdentity()
+    const { value: personas = EMPTY_LIST, loading } = useConnectedPersonas()
+
+    const { closeDialog } = useRemoteControlledDialog(PluginNextIDMessages.PersonaListDialogUpdated)
+
+    const { closeDialog: closeApplicationBoard } = useRemoteControlledDialog(
+        WalletMessages.events.ApplicationDialogUpdated,
+    )
+
+    const { value: currentPersonaIdentifier } = useAsync(() => {
+        return Services.Settings.getCurrentPersonaIdentifier()
+    }, [])
```

Comment:
It should revalidate due to settings changing.

```suggestion
    }, [currentPersonaIdentifier.value])
```